controllers/reviewsControllers.js

```diff
@@ -36,4 +36,19 @@ exports.getReviewsById = (request, response, next) => {
 		.catch((error) => {
 			next(error);
 		});
-};
\ No newline at end of file
+};
+
+exports.getCommentsByReview=(request,response,next)=>{
+	const {review_id}=request.params
+	const queryStringComments=format('SELECT * FROM comments ORDER BY created_at DESC')
```

Comment:
We do not want to be having SQL and query strings on the controller side. Anything related to this needs to be in the model function